### ISSUE TITLE
XML error when editing XML

### ISSUE THREAD

**Issue by Jessassin** [ATLauncher/Website-Issues#17]
I get the following error when saving XML for my pack:

XML Error
Error 1871: Element 'file': This element is not expected. Expected is ( folder ).
Line: 15
Column: 0

Line 15 (+) reads:

```
13    <delete>
14        <folder base="root" target="Flan%s%" />
15        <file base="root" target="updateLog.txt" />
16    </delete>
```

According to this wiki page: http://wiki.atlauncher.com/xml_guide:delete that config should work, but perhaps the error checking does not account for it.

**Comment by RyanTheAllmighty** [comment]
File may need to go before the folder tag possibly?

**Comment by Jessassin** [comment]
Indeed, putting the file line before folder did fix it. Still a bit odd, but it'll work. Thanks!

**Comment by RyanTheAllmighty** [comment]
It shouldn't matter, the validator is most likely borked so I'll look into it
